retro_data_structures/formats/mlvl.py

```diff
@@ -360,12 +361,13 @@ class AreaWrapper:
     _mrea: Mrea = None
     _strg: Strg = None
 
-    def __init__(self, raw: Container, asset_manager: AssetManager, flags: Container, names: Container, index: int):
+    def __init__(self, raw: Container, asset_manager: AssetManager, flags: Container, names: Container, index: int, parent_mlvl: Mlvl):
```

Comment:
If we pass the parent mlvl, there's no reason to pass the `flags` now, nor `asset_manager`. And I'd place the mlvl as first argument.

Maybe the names too, and we move the logic of picking the correct range to here? And maybe even `raw`.

Reply:
... you're absolutely right ofc but is it okay if i just slap a TODO/FIXME on this and do it later :sob: